src/js/components/Calendar/utils.js

```diff
@@ -150,7 +158,17 @@ export const normalizeForTimezone = (value, timestamp, normalize = true) => {
   let localOffset = 0;
   if (normalize) {
     if (timestamp && typeof timestamp === 'string') {
-      hourDelta = parseInt(timestamp?.split(':')[0], 10);
+      const day = Array.isArray(value) ? undefined : new Date(value);
+      const today = new Date();
+      if (
+        day &&
+        !inDaylightSavings(day) &&
+        day.getTimezoneOffset() > today.getTimezoneOffset()
+      ) {
+        hourDelta = parseInt(timestamp?.split(':')[0], 10) - 1;
```

Comment:
Rather than repeat the `parseInt()`, what about calculating `hourDelta` like before but then just adding or subtracting one from it if needed in the conditionals?